Migrate backend-requests from Sanity to MDX (#352)
clerkinc/clerk-docs: docs/backend-requests/handling/go.mdx

---
title: Handling requests with Go
description: Learn how to handle authenticated requests with Go middleware using Clerk.
---

# Handling requests with Go

## Go Middleware

> The Clerk Go SDK provides a simple middleware that adds the active session to the request’s context.

Reviewer: ```suggestion
The Clerk Go SDK provides an easy-to-use middleware that adds the active session to the request’s context.
```

Better to phrase it like that, otherwise people might feel bad if it's not "simple" for them

<InjectKeys>

```go filename=".env"
package main

import (
  "net/http"
  "github.com/clerkinc/clerk-sdk-go/clerk"
)

func main() {

> 	client, _ := clerk.NewClient(sk_test_••••••••••••••••••••••••••••••••••)

Reviewer 2: ```suggestion
	client, _ := clerk.NewClient({{secret}})
```
